StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentViewController.swift

```diff
@@ -168,13 +168,27 @@ class ConsentViewController: UIViewController {
             pane: .consent,
             analyticsClient: dataSource.analyticsClient,
             handleURL: { urlHost, nextPaneOrDrawerOnSecondaryCta in
-                if urlHost == "manual-entry" {
+                guard let urlHost, let address = StripeSchemeAddress(rawValue: urlHost) else {
```

Comment:
we can also get regular URLs that we open on the browser ([equivalent Android code](https://github.com/stripe/stripe-android/blob/8476a4de4c1deb52f3cb48ee56841aa9dd232da6/financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentViewModel.kt#L111)). Just want to make sure this new logic does not break those cases!

Reply:
This `AuthFlowHelpers.handleURLInTextFromBackend` method handles that for us behind the scenes! 

https://github.com/stripe/stripe-ios/blob/88c8c45053d7180c36ed92dd81e30b129ec36df5/StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/AuthFlowHelpers.swift#L51-L55